src/App.js

```diff
@@ -58,7 +60,7 @@ class ConnectedApp extends React.Component<{dispatch: Dispatch}, void> {
     // Setup auto-refresh.
     this.refreshIntervalId = setInterval(
       () => this.props.dispatch(requestStatus()),
-      5000,
+      60000,
```

Comment:
Can we do it only if some tests are failing and avoid doing it when everything is green?

Reply:
In any case the pollbot nginx server is caching for 30s so running it less than 30s after is useless.